### PR TITLE
Add First Design

### PR DESCRIPTION
**Pull request by alanluqman**
in this pull request I Added index.html file as a skeleton of the website and the style.css as styling file , then modified README.md file to give developers a brief about the website , also I Added linters to check code errors and MIT.md Added to files for Licensing .
Added Files :
- index.html
- style.css
- linters.yml
- MIT.md
- README.md

### CODE REVIEW

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @alanluqman , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 🔴  ⛔ 

You have submitted a project, that has no review.. please make sure you add some review inline comments and changes to be made. you are almost there 😄 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style/style.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+.nav-link {
+  text-decoration: none;
+  margin: 1rem 3rem;
+  display: inline-block;
+}
+
+.page-title {
+  font-size: 5rem;
```
Comment (alanluqman) [inline style/style.css:26]: - [ ]  I recommend you to use Arial font family for this Page Title .

File: index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <img src="image/logo.webp" alt="Company Logo" id="header-img" class="logo">
+      <nav id="nav-bar">
+        <a href="https://www.musicstore.de/en_DE/EUR" class="nav-link">Featuers</a>
+        <a href="https://www.musicstore.de/en_DE/EUR" class="nav-link">How It works</a>
+        <a href="https://www.musicstore.de/en_DE/EUR" class="nav-link">Prices</a>
+      </nav>
+    </header>
+    <h1 class="page-title">For inquiries</h1>  
+    <form method="post" class="form" action="https://www.musicstore.de/en_DE/EUR" id="form">
+      <input type="email" name="email" id="email" class="input" placeholder=" Enter email">
```
Comment (alanluqman) [inline index.html:21]: - [ ]  I feel that you need a specific Name for submitting a form ! So, Could you please Add another input field for Name.

**Review by alanluqman — COMMENTED** [review]
Well Done with creating this Web Page, It looks great. but there is a little  things I recommend you to change. Please Follow comments and good luck.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @alanluqman,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight
- [x] You've done a great job to address the issues a student needs to fix in a polite way and provided a reason it is important to fix them.

## Required Changes ♻️
- [ ] You should review your project documentation for it is missing some important part as the description, and explain to a student to make those changes.

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 3]
```diff
@@ -1 +1,57 @@
-# project-reviewing
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Review
+
+> This is one of Microverse project to practice github lessons,Create new branch and make a Pull Request (PR).
+
```
Comment (alanluqman) [inline README.md:6]: - [ ]  you had better to Add a clear Description For this website to make it more easy and understandable.

**Review by alanluqman — COMMENTED** [review]
Hi @alanluqman  

You did Great and there are nothing remains, only a bit of changes, So I recommend you to change it and your pull request will be approved. 

**Change Required**

**Review by zhadier — APPROVED** [review]
## Hi @alanluqman  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Optional
<hr/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**